src/main/scala/com/bwsw/cloudstack/entities/Executor.scala

```diff
@@ -29,13 +29,25 @@ import org.slf4j.LoggerFactory
 
 import scala.util.{Failure, Success, Try}
 
+/**
+  * Class is responsible for interaction with CloudStack
```

Comment:
Class provides functionality to interact with CloudStack

Reply:
ok